### ISSUE TITLE
How is it possible to get more tensors as an output for tf-serving with t2t-exporter?

### ISSUE THREAD

**Issue by JakubSido** [tensorflow/tensor2tensor#1722]
Hello, I am trying to use standard transformer_encoder for text2class task.
Right now, the trained model is running on tf-serving. How is it possible to add for example logits as another output and not only the class. 

Thank you
